src/julia_internal.h

```diff
@@ -368,20 +367,19 @@ static const int jl_gc_sizeclasses[] = {
     // pg = 2^14
     // sz = (div.(pg-8, rng).÷16)*16; hcat(sz, (pg-8).÷sz, pg .- (pg-8).÷sz.*sz)'
```

Comment:
pg = 2^16

seems like all of the totals below are off by 8?

Reply:
So I wasn't sure we counted the header, I did which I think makes more sense.